validator/engine/validator.js

```diff
@@ -4313,8 +4313,8 @@ function validateAttributes(
   const attrsByName = parsedTagSpec.getAttrsByName();
   for (const attr of encounteredTag.attrs()) {
     // For transformed AMP, attributes `class` and `i-amphtml-layout` are
-    // handled within validateSsrLayout.
-    if (context.isTransformed() &&
+    // handled within validateSsrLayout for non-sizer elements.
+    if (context.isTransformed() && encounteredTag.lowerName() !== 'i-amphtml-sizer' &&
```

Comment:
@Gregable @honeybadgerdontcare I'm not happy about hard-coding the sizer here. There's probably a better way...